libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/couchdb/lucene/LuceneAwareDatabaseConnector.java

```diff
@@ -49,6 +50,10 @@
     private static final Joiner AND = Joiner.on(" AND ");
     private static final Joiner OR = Joiner.on(" OR ");
 
+    private static final String LUCENE_ESCAPE_CHARS = "[\\\\+\\-\\!\\(\\)\\:\\^\\[\\]\\{\\}\\~\\*\\?\\&\\|\\\"]";
```

Comment:
Still an issue:
> Just for me to understand it, if I de-escape it one level (`\\` -> `\`) I get:
> ```
> "[\\+\-\!\(\)\:\^\]\{\}\~\*\?]";
> ```
> Which is a list of the escaped chars `+`, `-`, `!`, `(`, `)`, ... ,`?`, **with an additional slash at the beginning, which confuses me**. I would expect `"[\\\\\\+\\-...`

Reply:
`\\\\` -> `\\`

The leading slash is for escaping the backslash character in the string